sdk/keyvault/azure.keyvault.keys.txt

```diff
@@ -0,0 +1,180 @@
+**** Package/Module azure.keyvault.keys****
+class KeyClient(KeyVaultClientBase, object):
+	vault_url:str
+	credential
+
+	vault_url
+
+	def __init__(self, vault_url: str, credential, transport=None, api_version=None, **kwargs): ...
+	def backup_key(self, name: str, **kwargs) -> bytes: ...
+	def create_ec_key(self, name: str, hsm: bool, curve: curve=None, key_operations: key_operations=None, enabled: bool=None, expires: datetime.datetime=None, not_before: datetime.datetime=None, tags: dict=None, **kwargs) -> ~azure.keyvault.keys.models.Key: ...
+	def create_key(self, name: str, key_type: key_type, size: int=None, key_operations: key_operations=None, enabled: bool=None, expires=None, not_before: datetime.datetime=None, tags: dict=None, curve: curve=None, **kwargs) -> ~azure.keyvault.keys.models.Key: ...
```

Comment:
How many of the positional parameters can we make into positional parameters?

Reply:
Agree to fix. #7782 to track this